### PR TITLE
Fix molecule tests for scenario 'mtls-customcerts-rhel'

### PR DESCRIPTION
**Pull request by utkarsh5474**
# Description

Molecule tests for the scenario 'mtls-customcerts-rhel' were failing on task:
`TASK [confluent.zookeeper : Install the Zookeeper Packages]` with error as `Error: confluent-server conflicts with confluent-kafka-2.12-5.5.7-1.noarch`
Enabling default value of 'confluent_server_enabled' (which is true) resolved the issue. 

Fixes # ([As part of 5.5.x release readiness](https://confluentinc.atlassian.net/browse/ANSIENG-963?atlOrigin=eyJpIjoiYjZhOTEyNGUwYzgxNGRhMjllNGVkZmMzNWRlMDdiNWYiLCJwIjoiaiJ9))

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

`molecule --debug test -s mtls-customcerts-rhel` gives all pass/success.

**Test Configuration**: molecule run for scenario mtls-customcerts-rhel on 5.5.x

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

**Review by JumaX — CHANGES_REQUESTED** [review]
I don't think this is the correct solution here, as we have a few scenarios where we have deliberately set confluent_server_enabled to false as to test with Confluent Open Source version.

I would investigate the error further for resolution.

**Comment by utkarsh5474** [comment]
Yeah @JumaX 
I thought so too. 
But then, I tried to see the 6.0.x branch for this same scenario and file. The same fix came up sometime back. 
So, I think we'll need to evaluate the fix on 6.0.x too.

**Comment by utkarsh5474** [comment]
New changes: 
Undo previous change. Disable dockerfile use.

The custom Dockerfile.j2 installs confluent server on all images.
With `confluent_server_enabled = false`, the confluent-kafka-2.12 package is installed on all docker images. 
This is what is leading to the conflict.

`molecule --debug test -s mtls-customcerts-rhel` with these new changes gives all pass/success.

File: roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ platforms:
     groups:
       - zookeeper
     image: geerlingguy/docker-centos7-ansible
-    dockerfile: ../Dockerfile.j2
+    # dockerfile: ../Dockerfile.j2
```
Comment (JumaX) [inline roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml:10]: Please remove this line completely, it's not necessary if we don't use the docker file.

File: roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ platforms:
     groups:
       - kafka_broker
     image: geerlingguy/docker-centos7-ansible
-    dockerfile: ../Dockerfile.j2
+    # dockerfile: ../Dockerfile.j2
```
Comment (JumaX) [inline roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml:22]: Please remove this line completely, it's not necessary if we don't use the docker file.

File: roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml [hunk 3]
```diff
@@ -31,7 +31,7 @@ platforms:
...
```
Comment (JumaX) [inline roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml:34]: Please remove this line completely, it's not necessary if we don't use the docker file.

File: roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml [hunk 4]
```diff
@@ -43,7 +43,7 @@ platforms:
...
```
Comment (JumaX) [inline roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml:46]: Please remove this line completely, it's not necessary if we don't use the docker file.

File: roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml [hunk 5]
```diff
@@ -55,7 +55,7 @@ platforms:
     groups:
       - schema_registry
     image: geerlingguy/docker-centos7-ansible
-    dockerfile: ../Dockerfile.j2
+    # dockerfile: ../Dockerfile.j2
```
Comment (JumaX) [inline roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml:58]: Please remove this line completely, it's not necessary if we don't use the docker file.

File: roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml [hunk 6]
```diff
@@ -67,7 +67,7 @@ platforms:
     groups:
       - kafka_rest
     image: geerlingguy/docker-centos7-ansible
-    dockerfile: ../Dockerfile.j2
+    # dockerfile: ../Dockerfile.j2
```
Comment (JumaX) [inline roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml:70]: Please remove this line completely, it's not necessary if we don't use the docker file.

File: roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml [hunk 7]
```diff
@@ -79,7 +79,7 @@ platforms:
     groups:
       - kafka_connect
     image: geerlingguy/docker-centos7-ansible
-    dockerfile: ../Dockerfile.j2
+    # dockerfile: ../Dockerfile.j2
```
Comment (JumaX) [inline roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml:82]: Please remove this line completely, it's not necessary if we don't use the docker file.

File: roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml [hunk 8]
```diff
@@ -91,7 +91,7 @@ platforms:
     groups:
       - ksql
     image: geerlingguy/docker-centos7-ansible
-    dockerfile: ../Dockerfile.j2
+    # dockerfile: ../Dockerfile.j2
```
Comment (JumaX) [inline roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml:94]: Please remove this line completely, it's not necessary if we don't use the docker file.

File: roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml [hunk 9]
```diff
@@ -103,7 +103,7 @@ platforms:
     groups:
       - control_center
     image: geerlingguy/docker-centos7-ansible
-    dockerfile: ../Dockerfile.j2
+    # dockerfile: ../Dockerfile.j2
```
Comment (JumaX) [inline roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml:106]: Please remove this line completely, it's not necessary if we don't use the docker file.

**Review by JumaX — CHANGES_REQUESTED** [review]
Minor change, but great work in getting to the root cause!

**Review by JumaX — APPROVED** [review]
LGTM.

**Pull request merged**